Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -33,6 +33,9 @@
 import demistomock as demisto
 import warnings
 
+MAX_MSG_LENGTH = 60000
+MAX_CHARS_PREFIX_SUFFIX = 10000  # for shortening a long message
```

Comment:
Why two variables? I think we can simplifying it into just `MAX_ERROR_MESSAGE_LENGTH` and truncate `message=message[:MAX_ERROR_MESSAGE_LENGTH/2] + ...